### DISCUSSION TITLE
Steuern möglich

### DISCUSSION THREAD

**Discussion by SirUlbrich** [Tom-Bom-badil/samson_trovis_557x#51] [General (de/en)]
Hat es jemand geschafft oder weiss jemand wie ich den Heizkreislauf stoppen kann? Umwälzpumpe aus und Stellventil auf 0% zu fahren? Wie ich es an der Trovis von Hand mache verstehe ich. Über Modbus wäre es dennoch schöner.

**Comment by calle2010** [comment]
Betriebsart Rk1 (40106) auf den Wert 2 (Stand By) setzen sollte es tun.
Um wieder in den Automatikbetrieb zu wechseln, Coil 89 (Ebene Betriebsart Rk1) wieder "einschalten" (also auf 1 setzen). So geht es zumindest bei meiner 5573.
Für Rk2 entsprechend.
Die Pumpe geht erst nach etwas Verzögerung aus.

**Reply by SirUlbrich** [reply]
auf Standby klappte aus den Registern / Coils leider nur erkennbar dann auf 4 und 5 schalten zu können. Tag/Nachtbetrieb der Automatikbetrieb fehlte. Danke für die Info :)

**Comment by calle2010** [comment]
Das hat mich noch etwas beschäftigt: Was, wenn ich eine Automatik (wie Tag/Nachbetrieb) außer Kraft setze, und dann meine externe Steuerung ausfällt? 

Der Trovis 5573 (und vermutlich auch die anderen) haben dafür den Parameter "Leitsystemüberwachung" CO6 -> F07 - 1: Wenn 30 Minuten lang keine Modbus-Kommunikation erfolgt, werden alle Ebenenbits auf "autark" zurückgesetzt. 

Ich habe das gerade auch mal ausprobiert und den manuellen Tagbetrieb aktiviert. Dann habe ich mein Modbus-Interface abgeschaltet. Nach 30 Minuten war wieder das automatische Zeitprogramm aktiviert. Ich würde daher das Setzen von "CO6 -> F07" auf 1 zur Aktivierung der Leitsystemüberwachung als "best practice" ansehen, sobald ich etwas regelgesteuert von außen verändere.

@Tom-Bom-badil: Hast Du Dir schon einmal überlegt, solche Fragen irgendwo zusammenzufassen? Für mich war es z. B. nicht automatisch klar, wie ich mit den verschiedenen oben beschriebenen Parametern umgehen muss, um den gewünschten Effekt zu erzielen. Auch ist die Frage, ob man die Schlüsselzahl senden muss, immer wieder unklar. In diesen Fällen (Tag/Nachtbetrieb, Tag/Nacht-Solltemperatur) war das bei mir nicht notwendig. Aber ich habe auch den Drehregler nicht deaktiviert...

**Reply by Tom-Bom-badil** [reply]
Ja, so eine Art FAQ samt 'best practises' steht schon lange bei mir auf der ToDo-Liste. Ich bin mir nur nicht sicher, wo das am besten aufgehoben ist. Der richtige Ort wäre eigentlich das Wiki - aber etliche Fragen haben mir in der Vergangenheit gezeigt, dass dort Informationen einfach untergehen bzw. Aktualisierungen nicht wahrgenommen werden (siehe z.B. [hier](https://github.com/Tom-Bom-badil/samson_trovis_557x/wiki/Begriffe-und-'Basics'#n%C3%BCtzliches), oder auch darüber die Erklärungen, wie Modbus grundsätzlich funktioniert).

Vielleicht sollte ich einfach hier eine neue Kategorie dafür in den Discussions schaffen, und das Wiki auf die Selbstbauanleitung, das echte Grundlagenwissen sowie die Übersichten zu den Reglern reduzieren? Gern Vorschläge! :)

/tom
